src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { CopyToClipboard } from '../../shared/components/CopyToClipboard'
+import './styles.scss'
+import { formatPrice } from '../../shared/utils'
+
+interface Props {
+  data: {
+    NFTId: string
+    ledgerIndex: number | undefined
+    owner: string
+    isBurned: boolean | undefined
+    flags: string[] | undefined
+    transferFee: number
+    issuer: string
+    NFTTaxon: number
+    NFTSequence: number | undefined
+    uri: string | undefined
+    validated: boolean | undefined
+    status: string | undefined
+    warnings: string[] | undefined
+    minted: string | undefined
+    domain: string | undefined
+    emailHash: string | undefined
+  }
+}
+
+const Details = ({ data }: Props) => {
+  const { minted, domain, emailHash, NFTTaxon, uri, transferFee } = data
+  const { t } = useTranslation()
+
+  const formattedFee =
+    transferFee === 0
+      ? `${transferFee}%`
+      : `${formatPrice(transferFee / 1000, {
+          currency: 'none',
+          decimals: 5,
+          padding: 3,
+        })}%`
```

Comment:
why do we need to special case `0`? I don't think there's anything wrong with `0.0000%`